var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -44,10 +44,11 @@ class Googletest(CMakePackage):
     def cmake_args(self):
         spec = self.spec
         args = [
-            self.define_from_variant("gtest_disable_pthreads", "pthreads"),
             self.define_from_variant("BUILD_SHARED_LIBS", "shared"),
             self.define_from_variant("CMAKE_CXX_STANDARD", "cxxstd"),
         ]
+        if spec.satisfies("~pthreads"):
+            args.append(self.define("gtest_disable_pthreads", True))
```

Comment:
Append the argument whether pthreads is available or not
```suggestion
        args.append(self.define("gtest_disable_pthreads", !spec.satisfies("pthreads")))
```

Reply:
That's great!

Reply:
Whoops, sorry I've been coding in C++ too much recently. Should be:
```python
args.append(self.define("gtest_disable_pthreads", not spec.satisfies("pthreads"))
```

Reply:
In our case, this line didn't work since pthreads was never found regardless of how we specified pthreads in spack's install command.
We had to change your line to:
```Python
args.append(self.define("gtest_disable_pthreads", not spec.satisfies("+pthreads"))
```
(note the "+" before pthreads).

Reply:
@breyerml Want to submit a fix and ping me? Otherwise I can do that.

Reply:
I opened a new PR #44781.